spark/src/test/scala/org/apache/spark/sql/delta/MergeIntoSchemaEvolutionSuite.scala

```diff
@@ -62,54 +62,49 @@ trait MergeIntoSchemaEvolutionMixin {
       expectErrorWithoutEvolutionContains: String = null,
       confs: Seq[(String, String)] = Seq(),
       partitionCols: Seq[String] = Seq.empty): Unit = {
-    test(s"schema evolution - $name - with evolution disabled") {
-      withSQLConf(confs: _*) {
-        append(targetData, partitionCols)
-        withTempView("source") {
-          sourceData.createOrReplaceTempView("source")
 
-          if (expectErrorWithoutEvolutionContains != null) {
-            val ex = intercept[AnalysisException] {
-              executeMerge(s"delta.`$tempPath` t", s"source s", cond,
-                clauses.toSeq: _*)
-            }
-            errorContains(ex.getMessage, expectErrorWithoutEvolutionContains)
-          } else {
-            executeMerge(s"delta.`$tempPath` t", s"source s", cond,
-              clauses.toSeq: _*)
-            checkAnswer(
-              spark.read.format("delta").load(tempPath),
-              expectedWithoutEvolution.collect())
-            assert(
-              spark.read.format("delta").load(tempPath).schema.asNullable ===
-                expectedWithoutEvolution.schema.asNullable)
+    def executeMergeAndAssert(
```

Comment:
```suggestion
    private def executeMergeAndAssert(
```

Reply:
A `private` modifier is not allowed in a method within another method.
This method `executeMergeAndAssert` is private in nature.